index.html

```diff
@@ -1033,6 +1034,17 @@
       </p>
       <dl>
         <dt>
+          <dfn>paymentRequestId</dfn>
+        </dt>
+        <dd>
+          <a>paymentRequestId</a> is a freeform identifier for this
+          payment request. This field is intended to hold a unique
```

Comment:
Re: "intended", as it reads, it could be confused that the browser is supposed to use it for error recovery. I would remove the second sentence.